### PR TITLE
Config deployment

### PR DESCRIPTION
**Pull request by cjdenio**
This PR adds config deployment, and also moves some of the Dokku plugin files to the top-level (otherwise Dokku won't install the plugin)

### CODE REVIEW

File: .github/workflows/config_deploy.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+on:
+  push:
+    branches:
+      - master
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: mkdir -p ~/.ssh && echo "${{ secrets.SSH_PRIVATE_KEY }}" > ~/.ssh/id_rsa && chmod 600 ~/.ssh/id_rsa
```
Comment (anirudhb) [inline .github/workflows/config_deploy.yml]: This probably won't work, you want to use
```yaml
env:
  PRIVATE_KEY: ${{ secrets.PRIVATE_KEY }}
```

**Review by anirudhb — CHANGES_REQUESTED** [review]
lgtm. Can we also have dokkud auto-deploy, restart, and restart the haas app as well?

File: .github/workflows/config_deploy.yml (hunk 1 above)
Comment (cjdenio) [reply .github/workflows/config_deploy.yml]: I tested it, and it works perfectly

Comment (cjdenio) [reply .github/workflows/config_deploy.yml]: We shouldn't need to restart the HaaS app as long as the config deployment happens before the app deployment (which I can ensure)

**Review by anirudhb — APPROVED** [review]
lgtm

**Pull request merged**